packages/connect/src/api/common/MiscFees.ts

```diff
@@ -8,9 +8,9 @@ import type { CoinInfo, FeeLevel } from '../../types';
 export type Blocks = Array<string | undefined>;
 
 export const findBlocksForFee = (feePerUnit: string, blocks: Blocks) => {
-    const bn = new BigNumber(feePerUnit);
```

Comment:
I would prefer renaming chore changes to go in one commit and actual changes to logic to go into another one :D but I can live with it as it is :P